src/workloads/replication/startup/1_0_5GB.yml

```diff
@@ -0,0 +1,36 @@
+SchemaVersion: 2018-07-01
+Owner: mongodb/server-replication"
+Description: >
+  Loads the data for the light phase.
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 2500
```

Comment:
not sure if it makes a difference but do you actually  needs a maxPoolSize this large?

Reply:
Maybe try 300 as a starting point?

Reply:
Done